ci/ray_ci/container.py

```diff
@@ -58,7 +58,8 @@ def _run_tests_in_docker(test_targets: List[str], team: str) -> subprocess.Popen
             ]
         )
     commands.append(
-        "bazel test --config=ci $(./ci/run/bazel_export_options) "
+        "bazel test --config=ci --config=air-new-persistence-mode-off "
```

Comment:
you can just directly add `--test_env=RAY_AIR_NEW_PERSISTENCE_MODE=0` here, rather than defining a new config.

there is also the `bazel_export_options` thing.